src/modules/intelligence/domain/value_objects/translation_prompt.py

```
@@ -109,6 +109,54 @@ def render(self, target_language: str, tags: List[str]) -> 'TagTranslationPrompt
+        lang_name = _LANGUAGE_NAMES.get(target_language, target_language)
+        filled = self._content
+        filled = filled.replace("__TARGET_LANGUAGE__", lang_name)
+        filled = filled.replace("__TITLE__", title)
+        filled = filled.replace("__CONTENT__", content)
+        return ArticleBodyTranslationPrompt(_content=filled)
+
+    @staticmethod
+    def parse_response(text: str) -> tuple[Optional[str], Optional[str]]:
+        """Parse LLM response into (title, content) by section headers. Returns (None, None) on failure."""
+        import re
+        parts = re.split(r'\n(?=(?:Title|Content)\s*[:：]\s*)', text, flags=re.IGNORECASE)
```

Ruff (0.15.15)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)